lib/json_rules/cidr_rules.rb

```diff
@@ -46,8 +46,15 @@
  else (
         if (.Properties.SecurityGroupIngress|type == "array")
         then (
-               select(.Properties.SecurityGroupIngress[]|select(.CidrIp|endswith("/32")|not))
-             )
+               select(.Properties.SecurityGroupIngress[]|
+                      if (.CidrIp|type == "string")
+                      then (
+                          select(.CidrIp|endswith("/32")|not)
```

Comment:
i think the object bit above this needs this too.  thanks for the fix!  I will incorporate with a fresh test shortly